packages/core/src/components/Form.tsx

```diff
@@ -567,9 +567,23 @@ export default class Form<
     return getAllPaths(pathSchema);
   };
 
+  /** Returns the `formData` after filtering to remove any extra data not in a form field
+   *
+   * @param formData - The data for the `Form`
+   * @returns The `formData` after omitting extra data
+   */
+  omitExtraData = (formData: T | undefined): T | undefined => {
```

Comment:
Making the prop optional via the `?` notation has the same effect and is preferred

```suggestion
  omitExtraData = (formData?: T): T | undefined => {
```